packages/react-examples/.storybook/preview.js

```diff
@@ -61,7 +65,7 @@ function loadStories() {
   ];
 
   // @ts-ignore
-  if ('PACKAGE_NAME' === 'react') {
+  if ('PACKAGE_NAME' === 'react' || 'PACKAGE_NAME' === 'react-components') {
```

Comment:
`react-components` is a suite package which should show stories for all converged components.